service/lib/ops/userOps.js

```diff
@@ -492,6 +425,18 @@ const userOps = {
 
       cb()
     })
+  },
+
+  /**
+   * Delete user from teams
+   *
+   * @param  {String}   id
+   * @param  {Function} cb
+   */
+  deleteUserFromAllTeams: function deleteUserFromAllTeams (id, cb) {
```

Comment:
Moved it into [`test/utils.js`](https://github.com/nearform/labs-authorization/pull/276/files#diff-d183e88a1a4921431e4269373d7721f8R28) ;)